### PR TITLE
Remove unnecessary string()

### PR DESCRIPTION
**Pull request by tedyu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
PolicyPreferred and PolicyStrict are already string.

This PR removes the unnecessary string() around them in Name() func.

```release-note
NONE
```

### CODE REVIEW

**Review by lmdaly — APPROVED** [review]
LGTM

**Comment by tedyu** [comment]
@ConnorDoyle @derekwaynecarr 
Please take a look.

File: pkg/kubelet/cm/topologymanager/policy_preferred.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2015 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```
Comment (mattjmcnaughton) [inline pkg/kubelet/cm/topologymanager/policy_preferred.go]: Nit: I'm not sure we need to update the copyright when editing an already existing k8s file.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

modulo I'm not sure sure we need to update the copy right :)

File: pkg/kubelet/cm/topologymanager/policy_preferred.go (hunk 1 above)
Comment (tedyu) [reply pkg/kubelet/cm/topologymanager/policy_preferred.go]: this file was added this week

Comment (mattjmcnaughton) [reply pkg/kubelet/cm/topologymanager/policy_preferred.go]: :+1:

**Comment by tedyu** [comment]
ping @ConnorDoyle

**Comment by tedyu** [comment]
@sjenning
Can you take a look ?

**Comment by tedyu** [comment]
@ConnorDoyle
Please review.

**Review by vishh — APPROVED** [review]
/approve

**Pull request merged**